SparkPost/matchbox
# UX-159 Fix Button.Group alignment

### What Changed
- Fixes button group styling bugs

Before:
![Screen Shot 2020-05-08 at 9 43 37 AM](https://user-images.githubusercontent.com/3903325/81411717-c6db8180-9110-11ea-9128-fa1f6902f6f0.png)

After:
![Screen Shot 2020-05-08 at 9 43 21 AM](https://user-images.githubusercontent.com/3903325/81411716-c642eb00-9110-11ea-8214-b83681ddc73b.png)


### How To Test or Verify
- Run storybook with `npm run start:storybook`
- Visit the first tooltip story
- Verify alignment and margins are correct

### PR Checklist
